pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -250,13 +268,21 @@ func (plugin *kubenetNetworkPlugin) SetUpPod(namespace string, name string, id k
 
 	start := time.Now()
 	defer func() {
-		glog.V(4).Infof("TearDownPod took %v for %s/%s", time.Since(start), namespace, name)
+		glog.V(4).Infof("SetUpPod took %v for %s/%s", time.Since(start), namespace, name)
 	}()
 
 	pod, ok := plugin.host.GetPodByName(namespace, name)
 	if !ok {
 		return fmt.Errorf("pod %q cannot be found", name)
 	}
+	hostportMap, err := plugin.openPodHostports(pod)
```

Comment:
I don't think you can do this here.  You have to re-assert this in-line with the iptables creation.  You don't want to set an iptables rule for a port unless you are SURE that you were able to lock it.